.circleci/config.yml

```diff
@@ -208,7 +208,7 @@ jobs:
       - attach_workspace:
           at: .
       - run: |
-          export TAG=$(./dist/linux_amd64/circleci version)
+          export TAG=$(./dist/circleci-cli_linux_amd64/circleci version)
           sed -i -- "s/%CLI_VERSION_PLACEHOLDER%/$TAG/g" snap/snapcraft.yaml
```

Comment:
I'm wondering about how to make this fail – we could try this?

```yml
      - run:
           shell: /bin/bash -eo pipefail -o nounset
           command: |
             export TAG=$(./dist/circleci-cli_linux_amd64/circleci version)
             sed -i -- "s/%CLI_VERSION_PLACEHOLDER%/$TAG/g" snap/snapcraft.yaml
```

Reply:
Maybe? I'm not sure. How do you mean fail? I was already failing (red build).

Another way might be to define a regex for the CLI version. Then, after we pull the version here we can make sure it matches?